flow/packet-headers/arp.yaml

```diff
@@ -1,58 +1,78 @@
-openapi: 3.1.0
-
-info:
-  title: Arp packet header model
-  version: ^0.0.0
-
 components:
   schemas:
     Flow.Arp:
       description: ARP packet header
       type: object
       properties:
         hardware_type:
-          description: >-
-            Network link protocol type. 
-            Max length is 2 bytes.
-          $ref: './patterns.yaml#/components/schemas/Flow.Pattern'
+          x-pattern:
+            description: >-
+              Network link protocol type. 
+            format: hex
```

Comment:
Should these be modeled as `hex` or `integer`.  Thinking that in Python it is just as easy to specify hex values for these fields compared to strings.
`arp.hardware_type = 0xA`
vs 
`arp.hardware_type = "A"`